src/lib.rs

```diff
@@ -108,6 +108,9 @@ mod tests {
 		println!("Connecting to client 0...");
 		let mut client_0 = Endpoint::connect(&path).await
 			.expect("failed to open client_0");
+
+		tokio::time::delay_for(Duration::from_secs(2)).await;
```

Comment:
This is probably because server & client share the same task executor, and overlapped api in miow is not happy about this